gcore/utils.go

```diff
@@ -896,3 +905,22 @@ func toString(v any) (s string, typ dataTypeValidation) {
 	}
 	return ``, dtvNotString
 }
+
+func getIntEnvOrDefault(key string, defaultValue int) int {
+	envVal := os.Getenv(key)
+	if envVal == "" {
+		return defaultValue
+	}
+	val, err := strconv.Atoi(envVal)
+	if err != nil {
+		return defaultValue
+	}
+	return val
+}
+
+func GetConflictRetryConfig() ConflictRetryConfig {
+	return ConflictRetryConfig{
+		Amount:   getIntEnvOrDefault("TF_CONFLICT_RETRY_AMOUNT", ConflictRetryAmount),
```

Comment:
amount should be calculated based on 

```
Timeouts: &schema.ResourceTimeout{
			Create: schema.DefaultTimeout(5 * time.Minute),
			Delete: schema.DefaultTimeout(5 * time.Minute),
		},
```

Reply:
at the same time, timeouts should be increased to the 30 mins by default – for covered resources

Reply:
added. `Update` - what exactly do you mean? The one from loadbalancer resource (v1) - I think it should be marked as deprecated and only v2 allowed